nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPath.java

```diff
@@ -658,6 +693,46 @@ public boolean equals(Object obj) {
     }
   }
 
+  /**
+   * Represents all possible values that could be returned by calling {@code next()} on an {@code
+   * Iterator} variable
+   */
+  public static final class IteratorContentsKey implements MapKey {
+
+    /**
+     * Element for the local variable holding the {@code Iterator}. We only support locals for now,
+     * as this class is designed specifically for reasoning about iterating over map keys using an
+     * enhanced-for loop over a {@code keySet()}, and for such cases the iterator is always stored
+     * locally
+     */
```

Comment:
If we only support locals, is there a more precise type than `Element`?

Reply:
Yup, tightened to `VariableElement` in 76ddc4c